docs/query-builder.txt

```diff
@@ -176,15 +176,24 @@ Logical OR Example
 
 The following example shows how to chain the ``orWhere()``
 query builder method to retrieve documents from the
-``movies`` collection that either match the ``year``
-value of ``1955`` or match the ``title`` value ``"Back to the Future"``:
+``movies`` collection in which the value of the ``_id``
+field is ``ObjectId('573a1398f29313caabce9682')`` or
+the value of the ``title`` field is ``"Back to the Future"``:
 
 .. literalinclude:: /includes/query-builder/QueryBuilderTest.php
    :language: php
    :dedent:
    :start-after: begin query orWhere
    :end-before: end query orWhere
 
+.. note::
+
+   You can use the ``id`` alias in your queries to represent the
+   ``_id`` field in MongoDB documents, as shown in the preceding
+   code. When you run a find operation using the query builder, {+odm-short+}
+   automatically converts between ``id`` and ``_id``. Because of this behavior,
+   you cannot have two separate ``id`` and ``_id`` fields in your documents.
```

Comment:
We can explain that this behavior allows better compatibility with Laravel, "id" being the default name for primary keys in SQL databases.